qcodes/instrument_drivers/rigol/DS4000.py

```diff
@@ -250,3 +252,11 @@ def __init__(self, name, address, timeout=20, **kwargs):
 
         channels.lock()
         self.add_submodule('channels', channels)
+
+    def _check_firmware_version(self):
+        #Require version 00.02.03
+
+        idn = self.get_idn()
+        ver = [int(x) for x in idn['firmware'].split('.')]
+        if ver < [0,2,3]:
+            raise RuntimeError('Firmware version must be at least 00.02.03')
```

Comment:
Please add a clear explanation to the exception message about why the firmware needs to be at least that.